vars/capsuleSystemTests.groovy

```diff
@@ -47,6 +59,12 @@ void call(Map additionalConfig) {
   }
 
   stage('prepare') {
+    def publicIP = boxPublicIP()
+
+    print("The box public IP is: " + publicIP)
+    print("You may want to visit the nomad web interface: http://" + publicIP + ":4646")
```

Comment:
no tls?

Reply:
no. But I will add it when have 15 min today or in a few days

Reply:
We just want to see some logs